core/src/main/java/io/grpc/internal/ClientCallImpl.java

```diff
@@ -134,9 +135,18 @@ public void start(Listener<RespT> observer, Metadata headers) {
     }
```

Comment:
Hmm... the setting of this message encoding is now further away from the `stream.setCompressor()`. Unfortunate, but I guess it's okay.